.stylelintrc

```diff
@@ -0,0 +1,109 @@
+{
+  "rules": {
```

Comment:
I wasn't aware this existed, but let's extend it! I'll have to go through this again tomorrow to see what we need to override or append.